neurom/core/population.py

```diff
@@ -85,6 +89,16 @@ def _reset_cache(self):
         """Reset the internal cache."""
         self._files = [self._load_file(f) for f in self._files if f is not None]
 
+    @property
+    def mutable(self):
+        """Enable mixed tree processing if set to True."""
+        return self._mutable
+
+    @mutable.setter
+    def mutable(self, value):
+        self._mutable = value
+        self._reset_cache()
+
```

Comment:
This is not a responsibility of the wrapper class. Whether the inputs are mut/immut it should only depend on how the population is initialized.

Reply:
Since we can change the `process_subtrees` status of the morphologies after the `Population` is created, I don't see why we could not also change the `mutable` status of these morphologies?